### DISCUSSION TITLE
Activity Output issues after upgrading to 2.14.1

### DISCUSSION THREAD

**Discussion by pablotola** [elsa-workflows/elsa-core#5043] [Q&A]
I upgraded to version 2.14.1 yesterday and now my activity outputs are not working anymore, is this a breaking change or a bug?

This is a screenshot of the output log of one of my workflows working fine before and after the upgrade.

<img width="1049" alt="image" src="https://github.com/elsa-workflows/elsa-core/assets/2540082/46cc6554-3f2a-4d82-b1d7-9e1a205f52dd">

Project packages:

![image](https://github.com/elsa-workflows/elsa-core/assets/2540082/b581a2e5-2a9b-46e4-add0-9f003208fda4)

**Comment by glime-ai** [comment]
Based on the context provided, it seems that you are experiencing an issue after upgrading to Elsa version 2.14.1 where activity outputs are not working as they did prior to the upgrade. This could be either a breaking change introduced in the new version or a bug.

To determine whether it's a breaking change, you would typically look at the release notes for the version you upgraded to. Release notes often include information about any breaking changes, new features, and bug fixes. Unfortunately, the context provided does not include direct access to the release notes for version 2.14.1.

However, there have been instances mentioned in the context where users have encountered issues after upgrading Elsa versions, such as workflows not using the updated version until a restart (issue #4096) and errors with child workflows (issue #4830). These could be related to your issue if they involve changes to how activity outputs are handled.

If the release notes or documentation do not provide clarity, and you suspect this is a bug, the recommended course of action would be to report the issue on the Elsa GitHub repository's issues section. When reporting the issue, include all relevant details such as the version you upgraded from, the version you upgraded to, the expected behavior, the actual behavior, and any error messages or logs that can help diagnose the problem.

In the meantime, you can also check the GitHub issues for any similar reports from other users who might have experienced the same problem after upgrading to version 2.14.1. If such an issue already exists, you can follow it for updates or contribute to it with your findings.

Lastly, if the issue is causing significant disruption, you might consider rolling back to the previous version that was working for you until the issue is resolved.

Since the context does not provide direct links to the release notes or a list of breaking changes for version 2.14.1, I cannot provide a definitive answer on whether it's a breaking change or a bug. If you can provide the release notes or additional details, I may be able to offer more specific guidance.
